client/prototypes.h

```diff
@@ -117,12 +124,14 @@ TDNFRepoGetBaseUrl(
     const char* pszRepo,
     char** ppszBaseUrl
     );
+
 uint32_t
 TDNFRepoSetBaseUrl(
     PTDNF pTdnf,
-    const char* pszRepo,
-    char* pszBaseUrl
+    PTDNF_REPO_DATA_INTERNAL pszRepo,
```

Comment:
But if you see, TDNF uses the all CAPs notation for custom types (eg., `PTDNF_REPO_DATA_INTERNAL`). So metalinkfile_t and metalinkurs_t should actually be `TDNF_METALINK_FILES` and `TDNF_METALINK_URLS` etc.,

Reply:
Thanks, I followed same for the new structures.